automated install/basic-install.sh

```diff
@@ -2489,7 +2489,13 @@ main() {
 \\n\\nIPv4:	${IPV4_ADDRESS%/*}\
 \\nIPv6:	${IPV6_ADDRESS:-"Not Configured"}\
 \\nIf you have not done so already, the above IP should be set to static.\
-\\nView the web interface at http://pi.hole/admin:${WEBPORT} or http://${IPV4_ADDRESS%/*}:${WEBPORT}/admin\\n\\nYour Admin Webpage login password is ${pw}" "${r}" "${c}"
+\\nView the web interface at http://pi.hole/admin:${WEBPORT} or http://${IPV4_ADDRESS%/*}:${WEBPORT}/admin\\n\\nYour Admin Webpage login password is ${pw}\
+\\n
+\\n
+\\nTo allow your user to use all CLI functions without authentication,\
+\\nadd your user to the 'pihole' group with something like:\
```

Comment:
```suggestion
\\nadd your user to the 'pihole' group with:\
```

"Something like" is a bit too non-specific.

Also, there is some similar output to the CLI - possibly worth putting it there, too.

Reply:
Done.